doc/content/integrations/ifttt/_index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "IFTTT"
+description: ""
+weight: 
+---
+
+[IFTTT](https://ifttt.com/) is a service that creates chains of conditional statements, called applets, that can be triggered by the events occurring within various web services. These applets connect multiple applications in order to run automated tasks.
+
+<!--more-->
+
+At the time of writing, the implementation of this integration with only [HTTP Webhooks](https://thethingsstack.io/integrations/webhooks/) is not fully achievable. Hence, [Node-RED](https://nodered.org/) will be used a part of the setup.
```

Comment:
Might be better not to include this. What about something like this:

For this integration to work, Node-RED will be used to connect The Things Stack with IFTTT.